src/core/server/http/http_server.ts

```diff
@@ -340,7 +340,9 @@ export class HttpServer {
       const parentContext = executionContext?.getParentContextFrom(request.headers);
       if (parentContext) executionContext?.set(parentContext);
 
-      executionContext?.setRequestId(requestId);
+      if (requestId) {
+        executionContext?.setRequestId(requestId);
+      }
```

Comment:
@mshustov not 100% how the `AsyncLocalStorage` is working. Is that fine, or should we call `setRequestId` with `undefined` instead?